### ISSUE TITLE
uirevision equal to 0 does not preserve ui state in dcc.Graph

### ISSUE THREAD

**Issue by pmkroeker** [plotly/dash-core-components#956]
Not sure if this belongs here or in plotly, so let me know if this should be moved.

I was trying to add uirevision to preserve legend selections on a barchart, so I set `uirevision=0` and left it unchanged. The chart continued to reset the selections back to the default when the chart was updated.

However, if the value of `uirevision` is changed to 1, and left unchanged, then it works as expected.

According to [this post](https://community.plotly.com/t/preserving-ui-state-like-zoom-in-dcc-graph-with-uirevision-with-dash/15793/3), `uirevision` can be set to any value, and the chart should only reset the ui state when the value is changed.

**Comment by pmkroeker** [comment]
I'm not sure if this should be marked as a bug, or more of a gotcha that people should be aware of.

**Comment by alexcjohnson** [comment]
Yes, we should document better that `uirevision` needs to be truthy to be active. Thanks @pmkroeker !
